tests/manage/mcg/test_bucket_creation.py

```diff
@@ -178,3 +180,27 @@ def test_duplicate_bucket_creation(
                 logger.info(
                     f"Create duplicate bucket {bucket_name} failed as" " expected"
                 )
+
+    @bugzilla("2179271")
+    @pytest.mark.parametrize(
+        argnames="amount,interface",
+        argvalues=[
+            pytest.param(*[1, "OC"], marks=[tier1]),
+        ],
+    )
+    def test_check_for_error_in_rook_op_logs(self, bucket_factory, amount, interface):
+        """
+        Test to check for error 'malformed BucketHost "s3.openshift-storage.svc"
+        in rook operator logs post creation of noobaa obc
+
+        """
+
+        bucket_factory(amount, interface)
```

Comment:
In the test, we are creating only one OBC. Are we sure that the error message appears in the logs during every OBC creation?
I see below comment in the BZ, 
`+ It has around 16 OBC's from NooBaa and RGW, but the error is only for 2 OBB's.`

Reply:
The test failed in 4.12 (As expected) , from the logs I could see that even 1 OBC creation leads to that error.
http://pastebin.test.redhat.com/1104678

Reply:
Thanks for testing this test on 4.12 as we discussed.